.github/workflows/test-driver.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Suggestion: You can also add `workflow_dispatch` which will allow users to manually trigger GHA from the actions tab.

```
on:
  pull_request:
    branches: [ master ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
```